### PR TITLE
Add a column to show hosting server in the task list

### PR DESCRIPTION
**Pull request by Lqp1**
JIRA: MESOS-4875

ex

### CODE REVIEW

File: src/js/components/TaskListItemComponent.jsx [hunk 1]
```diff
@@ -300,6 +301,9 @@ var TaskListItemComponent = React.createClass({
           <br />
           {endpoints}
         </td>
+        <td>
+          {host}
```
Comment (warrenseine) [inline src/js/components/TaskListItemComponent.jsx]: Would it be interesting to link this to https://consul-info-html.par.preprod.crto.in/consul-nodes-ui.html?node_filter=mesos-slave274 ?

Comment (Lqp1) [reply src/js/components/TaskListItemComponent.jsx]: I'll check if this is easy to do; if not I'll do it in another review :)

Comment (Lqp1) [reply src/js/components/TaskListItemComponent.jsx]: Done; thanks for the feedback! :)

**Review by fquesnel — APPROVED** [review]
Lgtm

**Review by komuta — APPROVED** [review]
LGTM 🙈

**Pull request closed without merging**